modules/nixos/holo-deploy.nix

```diff
@@ -10,4 +14,16 @@
       description = "IP addres or host name to connect to the host";
     };
   };
+
+  config = {
+    environment.etc."nix/sourceInfo.json" = {
+      enable = true;
+      text = builtins.toJSON (builtins.removeAttrs self.sourceInfo ["outPath"]);
```

Comment:
if the outPath attribute isn't removed the file would _only_ contain the outPath i.e. nix-store path of the sources, which isn't helpful in this use-case.